README.md

```diff
@@ -14,8 +14,15 @@ be easy to add however.
 
 # Usage
 
-## Install Requirements
-```pip install -r requirements.txt```
+## Installation
+
+```
+mkdir venv
+virtualenv venv
+. venv/bin/activate
+pip install -r requirements.txt
```

Comment:
This also should not be needed as `python setup.py install` should install the requirements found in the setup.py file

Reply:
ditto